examples/python/quickstart_sql_on_paths.py

```diff
@@ -0,0 +1,74 @@
+
+from pyspark.sql import Column, DataFrame, SparkSession, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import threading
+import tempfile
+import os
+
+table_dir = "/tmp/delta-table"
+# Clear any previous runs
+try:
+    shutil.rmtree(table_dir)
+except:
+    pass
+
+# Enable SQL/DML commands and Metastore tables for the current spark session.
+# We need to set the following configs 
+
+spark = SparkSession.builder \
+    .appName("quickstart") \
+    .master("local[*]") \
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
+    .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
+    .getOrCreate()
+
+# Clear any previous runs
+spark.sql("DROP TABLE IF EXISTS newData")
```

Comment:
shouldn't we delete the path based table as well ?

Reply:
i wasnt dropping it because i was leaving it for the user explore after running this example.